src/components/avatar/Avatar.scss

```diff
@@ -54,6 +68,19 @@
         .avatar-initials {
             font-size: 11px;
         }
+
+        &.avatar--isExternal {
+            .avatar-external-badge {
+                width: 16px;
```

Comment:
minor comment: just not sure why we have a bigger sizing for a small avatar? why not just use the above class styling?

Reply:
this accounts for the total size, including the border. due to how the asset sizing is calculated, and the design requirements, we want the asset to have 14px size with 2px border